src/PrizeVault.sol

```diff
@@ -781,18 +789,36 @@ contract PrizeVault is TwabERC20, Claimable, IERC4626, ILiquidationSource, Ownab
     /// @dev The yield fee balance is included since it's cheaper to keep track of those shares
     ///      internally instead of doing an additional TWAB mint on every liquidation.
     /// @param _totalSupply The total share supply of the vault
+    /// @param _yieldFeeBalance The unrealized yield fee balance
     /// @return The total asset debt of the vault
-    function _totalDebt(uint256 _totalSupply) internal view returns (uint256) {
-        return _totalSupply + yieldFeeBalance;
+    function _totalDebt(uint256 _totalSupply, uint256 _yieldFeeBalance) internal pure returns (uint256) {
+        return _totalSupply + _yieldFeeBalance;
     }
 
     /// @notice Returns the remaining supply that can be minted without exceeding the TwabController limits.
     /// @dev The TwabController limits the total supply for each vault to uint96
+    /// @dev The yield fee balance is included to account for the unrealized shares
     /// @param _totalSupply The total share supply of the vault
+    /// @param _yieldFeeBalance The unrealized yield fee balance
     /// @return The remaining supply that can be minted without exceeding TWAB limits
-    function _twabSupplyLimit(uint256 _totalSupply) internal pure returns (uint256) {
+    function _twabSupplyLimit(uint256 _totalSupply, uint256 _yieldFeeBalance) internal pure returns (uint256) {
```

Comment:
I feel like this function name could use a tweak for clarity....

The `@notice` says "Returns the remaining supply that can be minted without exceeding the TwabController limits"

calling it "_mintLimit" feels clearer to me, and more aligned with the docs

because the twab supply limit is really type(uint96).max